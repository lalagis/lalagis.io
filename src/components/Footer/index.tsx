export default function Footer() {
  return (
    <footer className="mt-12">
      <p className="text-sm op-40">
        Lulala Â© 2023
      </p>
    </footer>
  )
}