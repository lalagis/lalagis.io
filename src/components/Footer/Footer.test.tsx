import { describe, expect, test } from "vitest"
import renderer from 'react-test-renderer'
import Footer from "."

describe('Footer', () => {
  test('load footer', () => {
    const component = renderer.create(<Footer />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})