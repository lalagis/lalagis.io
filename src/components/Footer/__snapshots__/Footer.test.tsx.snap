// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > load footer 1`] = `
<footer
  className="mt-12"
>
  <p
    className="text-sm op-40"
  >
    Lulala Â© 2023
  </p>
</footer>
`;
